WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingBottomSheetFragment.kt

```diff
@@ -80,6 +84,54 @@ class PrepublishingBottomSheetFragment : WPBottomSheetDialogFragment(),
         }
     }
 
+    private fun setupFragmentContainerHeightBasedOnDevice() {
```

Comment:
@malinajirka At first, this layout had no defined height. I utilized `wrap_content` for the `fragment container` and it worked well until I began navigating to other screens that ended up having different heights from the home screen. So navigating from one screen to the next would include a height change and cause a flicker and design inconsistency. 

1. I decided to utilized the drawn height of the `fragment container` when the home screen is drawn and set that as the permanent height overriding the `wrap_content` this worked well but when the device rotated/ app killed it was lost, so I decided to persist this height. Overall, I didn't like that solution and there were some times when the height would still be reset due to different view configuration changes so I opted against it. 

2. I then tried simply setting a single height on the bottom sheet but this caused the UI to look okay on some screens and not properly sized on others so I decided against it. 

3. So below, is the solution that seemed most workable for the different screen sizes. The height of the container is calculated dynamically based on the orientation and device. The main issue I have noticed is that sometimes there's a bit of extra padding at the bottom because the formula might give a bottom sheet a bit more height than it needs. 

4. Also, I was wondering if I should be utilizing screen-specific layouts to host this logic as that would make it clearer how the UI adapts to each screen.  I opted from doing this, to gain feedback. 

Let me know what your thoughts on this are.

Reply:
Thanks! I tried the shared element and I didn't record a gif because it was really bad 😅 For now, I removed this logic and set a minimum height 2deda1d c2c9837